contrib/opencensus-ext-datadog/opencensus/ext/datadog/traces.py

```diff
@@ -0,0 +1,372 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import defaultdict
+import codecs
+from datetime import datetime
+import bitarray
+from opencensus.common.transports import sync
+from opencensus.trace import base_exporter
+from opencensus.trace import span_data
+from opencensus.ext.datadog.transport import DDTransport
```

Comment:
Please sort your imports.

Reply:
I am use to go where first imports are grouped (stdlib, 3rd party, internal) then sorted is the convention for Python the same, or is it just sorted as a unit?